src/anyp/PortCfg.cc

```diff
@@ -111,3 +111,9 @@ AnyP::PortCfg::detailCodeContext(std::ostream &os) const
     return os;
 }
 
+std::ostream &
+AnyP::PortCfg::print(std::ostream &os) const
+{
+    return os << s << " " << flags;
```

Comment:
We should decide what other PortCfg parts need displaying.

Reply:
> usually better to report that name than to report the listening address 

Done. As far as PortCfg::name always has some meaningful value (the explicit name or, if unspecified, the listening address by default) it should be enough to print this field only. The 'else' part in :PortCfg::print() is unreacheable and  'just in case'.